apps/admin/src/actions/composables/mutations/account/useUpdateAccountMutation.ts

```diff
@@ -48,9 +48,11 @@ export const useUpdateAccountMutation = (): ResolvedQuery<BackendEndpoint.Update
         const translation = 'error_invalid_api_key';
 
         notificationStore.add({
-          category: NotificationCategory.Api,
           content: translate(`${translation}_content`),
           title: translate(translation),
+          tags: {
+            category: NotificationCategory.Api,
```

Comment:
category hoort niet in tags, dit is meer een `source` oid. (dan moet je de enum ook hernoemen)